resources/qml/MatrixText.qml

```diff
@@ -29,8 +31,32 @@ TextEdit {
         id: ma
 
         anchors.fill: parent
-        acceptedButtons: Qt.NoButton
+        acceptedButtons: {
+            if (message.selectedText == "")
+                return Qt.NoButton;
+            else
+                return Qt.RightButton;
+        }
+
         cursorShape: parent.hoveredLink ? Qt.PointingHandCursor : Qt.ArrowCursor
+
+        onClicked: {
+            if (parent.selectedText != "")
+                copyMenu.popup();
+        }
+    }
+
+    Menu {
```

Comment:
Don't create a menu per message, it is horrible for performance!

Reply:
(I made that mistake before)

Reply:
OK